src/android/FirebaseInstanceIdRouterService.java

```diff
@@ -0,0 +1,20 @@
+package com.zenput.push;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import io.intercom.android.sdk.push.IntercomPushClient;
+import com.pushwoosh.PushwooshFcmHelper;
+
+public class FirebaseInstanceIdRouterService extends FirebaseMessagingService {
```

Comment:
since this class is also a subclass of `FirebaseMessagingService `, we don't need a whole new file for it. Please remove to the other java file